spec/lib/utils/task-option-validator-spec.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2016, EMC
+
+'use strict';
+
+describe("TaskOption Validator", function () {
+    var taskOptionValidator;
+    var testSchema1 = { 
+        properties: {
+            repo: {
+                type: 'string',
+                format: 'uri'
+            }
+        }
+    };
+
+    before(function() {
+        helper.setupInjector([
+            helper.require('/lib/utils/task-option-validator')
+        ]);
+        taskOptionValidator = helper.injector.get('TaskOption.Validator');
+    });
+
+    describe('register', function() {
+        it('should load all JSON schemas under /lib/task-data/schemas', function() {
+            var schemaId = 'rackhd/schemas/v1/tasks/install-os-common';
```

Comment:
Suggest removing the dependency here. You can stub the FileLoader and let it return some example schema files.